src/styles/colors.ts

```diff
@@ -0,0 +1,97 @@
+import {Color} from './themes/ThemeColors';
+
+/**
+ * DO NOT import colors.js into files. Use ../themes/default.js instead.
+ */
+const colors: Record<string, Color> = {
+    black: '#000000',
```

Comment:
Should we add a comment for this block of colors like it was added for the other ones?